### PR TITLE
Remove env tf var

### PR DESCRIPTION
**Pull request by JamesGardiner**

### CODE REVIEW

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Need to remove env from `tfvars.example`

Also, I think we need to remove the use of ENV all together, we have an `ENV` environment variable in the `create.sh` and `destroy.sh` which we should change to `EXISTING_PROJECT_ID`. This would also make the [concourse](https://github.com/ONSdigital/census-eq-concourse/pull/29) changes simpler. ENV was only needed when we wanted to provision project that we didn't know the ID to, but for this case, the ID is always known.

```tf
#!/usr/bin/env bash

set -ex

if [[ -z "$EXISTING_PROJECT_ID" ]]; then
  echo "Missing EXISTING_PROJECT_ID variable"
  exit 1
fi

TERRAFORM_STATE_BUCKET="${TERRAFORM_STATE_BUCKET:-census-eq-monitoring-tfstate}"

terraform init --upgrade --backend-config prefix=${EXISTING_PROJECT_ID} --backend-config bucket=${TERRAFORM_STATE_BUCKET}

# Roll out infrastructure
terraform apply -auto-approve

echo
echo "Successfully provisioned monitoring and alert policies"
```

Would be run using:
`EXISTING_PROJECT_ID=census-eq-mebin-v3-33511a76 ./create.sh`

**Comment by JamesGardiner** [comment]
@MebinAbraham ENV is now removed entirely

**Comment by JamesGardiner** [comment]
[GCP Stackdriver Alerts - logging metrics, monitoring and alert policies](https://trello.com/c/Amq6k0Az/3255-gcp-stackdriver-alerts-logging-metrics-monitoring-and-alert-policies)

**Comment by JamesGardiner** [comment]
> Need to remove env from `tfvars.example`
> 
> Also, I think we need to remove the use of ENV all together, we have an `ENV` environment variable in the `create.sh` and `destroy.sh` which we should change to `EXISTING_PROJECT_ID`. This would also make the concourse changes simpler. ENV was only needed when we wanted to provision project that we didn't know the ID to, but for this case, the ID is always known.
> 
> ```terraform
> #!/usr/bin/env bash
> 
> set -ex
> 
> if [[ -z "$EXISTING_PROJECT_ID" ]]; then
>   echo "Missing EXISTING_PROJECT_ID variable"
>   exit 1
> fi
> 
> TERRAFORM_STATE_BUCKET="${TERRAFORM_STATE_BUCKET:-census-eq-monitoring-tfstate}"
> 
> terraform init --upgrade --backend-config prefix=${EXISTING_PROJECT_ID} --backend-config bucket=${TERRAFORM_STATE_BUCKET}
> 
> # Roll out infrastructure
> terraform apply -auto-approve
> 
> echo
> echo "Successfully provisioned monitoring and alert policies"
> ```
> 
> Would be run using:
> `EXISTING_PROJECT_ID=census-eq-mebin-v3-33511a76 ./create.sh`

Done

File: README.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ Logging metric based monitoring and alert policies must first be provisioned to
 
 Rename `terraform.tfvars.example` to `terraform.tfvars` and fill in the values. (Ask a team member for help).
 
-Create with `ENV=your-env ./create.sh`.
+Create with `EXISTING_PROJECT_ID=your-env ./create.sh`.
```
Comment (MebinAbraham) [inline README.md]:
```suggestion
Create with `EXISTING_PROJECT_ID=your-project-id ./create.sh`.
```

File: README.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ Logging metric based monitoring and alert policies must first be provisioned to
...
 
-Destroy with `ENV=your-env ./destroy.sh`
+Destroy with `EXISTING_PROJECT_ID=your-env ./destroy.sh`
```
Comment (MebinAbraham) [inline README.md]:
```suggestion
Destroy with `EXISTING_PROJECT_ID=your-project-id ./destroy.sh`
```

**Review by MebinAbraham — APPROVED** [review]
👍

**Comment by MebinAbraham** [comment]
Needed for: https://github.com/ONSdigital/census-eq-concourse/pull/29

**Review by pricem14pc — APPROVED** [review]

**Pull request merged**